### ISSUE TITLE
No file in Output directory

### ISSUE THREAD

**Issue by cqbaoyi** [numat/RASPA2#24]
Hi! I have been using RASPA2 for a while.
Recently I installed it on another Linux platform. When I simulate a MC job, it runs well except that there is no file created in the Output/System_0 directory. No matter what parameters I try, this bug persists.
How could I fix it? Thanks.

**Comment by patrickfuller** [comment]
I'd recommend emailing David Dubbeldam (the author of RASPA) directly.

**Comment by lhchou** [comment]
@cqbaoyi  Did you solved the problem? I came across the same problem, could you help me?

**Comment by cqbaoyi** [comment]
@lhchou No. Such a problem occurs unpredictably. I would consider it a unfixed bug.

**Comment by lhchou** [comment]
@cqbaoyi Well, thanks.

**Comment by Stees** [comment]
I have also experienced this problem.
